client/interfaces.go

```diff
@@ -444,6 +444,9 @@ type StoragePoolVolumeCopyArgs struct {
 
 	// API extension: storage_api_volume_snapshots
 	VolumeOnly bool
+
+	// New project for the target
+	TargetProject string
```

Comment:
This shouldn't be needed, Copy in the client package is handled by passing a `source` and `destination` InstanceServer, those can be using different projects.

For example of how we handle this with instances, look for `req.Source.Project = sourceInfo.Project` in that file for how we compare the source and destination and check if the projects differ.